challenges/intermediate/crack-caesar-cipher/solution/fdb86/caesar-cipher.ts

```diff
@@ -0,0 +1,21 @@
+export function encode(word: string, shift: number) {
+
+  shift = shift % 26;
```

Comment:
Please instantiate a new variable instead of modifying the function argument

Reply:
ok new const created